.clang-format

```diff
@@ -13,8 +13,8 @@ TabWidth:        4
 IndentWidth:     4
 ConstructorInitializerIndentWidth: 4
 
-IndentPPDirectives: AfterHash
-PPIndentWidth:   2
+#IndentPPDirectives: AfterHash
+#PPIndentWidth:   2
```

Comment:
This needs to be restored or changed to `BeforeHash`. See https://github.com/irods/irods/pull/6647#issuecomment-1279768353

Reply:
Reverted changes that attempted to disable PP formatting.

Reply:
Leaving `AfterHash` as the default. Developers can choose how they want to format PP directives using clang-format off/on.

Reviewers will will the approval regarding formatting.